CHANGELOG.md

```diff
@@ -0,0 +1,117 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/localheinz/composer-normalize/compare/0.9.0...HEAD) ([`0.9.0...master`](https://github.com/localheinz/composer-normalize/compare/0.9.0...master))
+
+### Added
+
+*
```

Comment:
is it autogenerated?
if not, i would suggest to skip empty sections,

even keep a changelog skips empty sections in it's own changelog

Reply:
Not (yet) autogenerated, just left them here so I have the right order.

Reply:
keep the order and simply leave filled points.

eg, for bugfix releases, you won't have any features, right ? no point to always have them present

Reply:
Thank you, @keradus!